examples/seismic/tutorials/06_skimage_tv.ipynb

```diff
@@ -0,0 +1,416 @@
+    "            return self.__add__(other)\n",
+    "\n",
+    "from devito import Function, clear_cache\n",
+    "\n",
+    "# Create FWI gradient kernel for a single shot\n",
+    "def fwi_gradient_i(param):\n",
+    "    # Need to clear the workers cache.\n",
+    "    clear_cache()\n",
+    "\n",
+    "    # Load the current model and the shot data for this worker.\n",
+    "    # Note, unlike the serial example the model is not passed in\n",
+    "    # as an argument. Broadcasting large datasets is considered\n",
+    "    # a programming anti-pattern and at the time of writing it\n",
+    "    # it only worked relaiably with Dask master. Therefore, the\n",
+    "    # the model is communicated via a file.\n",
+    "    model0 = get_current_model(param)\n",
+    "    src, rec, nt, solver = get_data(param)\n",
+    "    \n",
+    "    # Create symbols to hold the gradient and the mismatch between\n",
+    "    # the 'measured' and simulated data.\n",
+    "    grad = Function(name=\"grad\", grid=model0.grid)\n",
+    "    residual = Receiver(name='rec', grid=model0.grid, ntime=nt, coordinates=rec.coordinates.data)\n",
+    "    \n",
+    "    # Compute simulated data and full forward wavefield u0\n",
+    "    d, u0, _ = solver.forward(src=src, m=model0.m, save=True)\n",
+    "        \n",
+    "    # Compute the data mismatch (residual) and objective function  \n",
+    "    residual.data[:] = d.data[:] - rec.data[:]\n",
+    "    f = .5*np.linalg.norm(residual.data.flatten())**2\n",
+    "    \n",
+    "    # Compute gradient using the adjoint-state method. Note, this\n",
+    "    # backpropogates the data mismatch through the model.\n",
+    "    solver.gradient(rec=residual, u=u0, m=model0.m, grad=grad)\n",
+    "    \n",
+    "    # Taking here to avoid a (probably overzealous) destructor deleting\n",
+    "    # the gradient before Dask has had a chance to communicate it.\n",
+    "    g = np.array(grad.data[:])\n",
+    "    \n",
+    "    # return the objective functional and gradient.\n",
+    "    return fg_pair(f, g)\n",
+    "\n",
+    "import numpy as np\n",
+    "from distributed import Client\n",
+    "\n",
+    "# Dumps the model to disk; workers will pick this up when they need it.\n",
+    "def dump_model(param, model):\n",
+    "    np.save(param['model'], model.astype(np.float32))\n",
+    "\n",
+    "def fwi_gradient(model, param):\n",
+    "    # Dump a copy of the current model for the workers\n",
+    "    # to pick up when they are ready.\n",
+    "    param['model'] = \"model_0.npy\"\n",
+    "    dump_model(param, model)\n",
+    "\n",
+    "    # Define work list\n",
+    "    work = [dict(param) for i in range(param['nshots'])]\n",
+    "    for i in  range(param['nshots']):\n",
+    "        work[i]['shot_id'] = i\n",
+    "        \n",
+    "    # Start dask cluster\n",
+    "    client = Client()\n",
+    "    \n",
+    "    # Map worklist to cluster\n",
+    "    fgi = client.map(fwi_gradient_i, work)\n",
```

Comment:
"Map worklist to cluster" -> although this is what is happening here, strictly speaking, the word map in `client.map` does not mean "mapping worklist to cluster". The word map here refers to a concept from [functional programming](https://en.wikipedia.org/wiki/Map_(higher-order_function)) and you might be at the risk of confusing someone who comes in with that background. What you are asking for in this line is a mapping from a list of inputs to a list of outputs. That it involves a cluster of workers is actually a detail abstracted away by dask.

Reply:
changed comment to:  Distribute worklist to workers.